### PR TITLE
Clarifications and fixes in GMSA doc

### PR DESCRIPTION
**Pull request by ddebroy**
Addressed some review comments from @michmike at https://github.com/kubernetes/website/pull/12936#pullrequestreview-215338457 onwards.

/assign @jimangel @michmike

### CODE REVIEW

**Comment by michmike** [comment]
hey @ddebroy even though the changes in markdown look great, when i view this in netlify it is broken..
https://5c8e0300fe71c700097753dc--kubernetes-io-vnext-staging.netlify.com/docs/tasks/configure-pod-container/configure-gmsa/

image

**Comment by JeremyWx** [comment]
> hey @ddebroy even though the changes in markdown look great, when i view this in netlify it is broken..
> https://5c8e0300fe71c700097753dc--kubernetes-io-vnext-staging.netlify.com/docs/tasks/configure-pod-container/configure-gmsa/

@michmike and @ddebroy it looks like we need to add the ```{{% capture prerequisites %}}``` and ```{{% capture steps %}}``` tags to make it conform?

**Comment by ddebroy** [comment]
@michmike thanks for pointing out the errors above in the rendering! 
@JeremyWx and I have fixed the formatting issues. PTAL.

**Comment by michmike** [comment]
I will tomorrow AM.

Sent from TypeApp<http://www.typeapp.com/r?b=14670>
On Mar 18, 2019, at 1:44 AM, Deep Debroy <notifications@github.com<mailto:notifications@github.com>> wrote:

@michmike<https://github.com/michmike> thanks for pointing out the errors above in the rendering!
@JeremyWx<https://github.com/JeremyWx> and I have fixed the formatting issues. PTAL.

**Comment by michmike** [comment]
thank you for all the changes and the updated rendering
/lgtm

File: content/en/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 1]
```diff
@@ -191,4 +196,10 @@ spec:
         beta.kubernetes.io/os: windows
 ```
 
+### Conclusion
```
Comment (jimangel) [inline content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: Following the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice), we should use an active voice. (ex: The mutating webhook will resolve -> The mutating webhook resolves... for all 3 examples)

**Comment by ddebroy** [comment]
Addressed comment around "will" -> active voice above.

File: content/en/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 2]
```diff
@@ -193,7 +193,7 @@ spec:
         beta.kubernetes.io/os: windows
 ```
 
-As pod specs with GMSA annotations (as described above) are applied in a cluster configured for GMSA, the following sequence of events will take place:
+As pod specs with GMSA annotations (as described above) are applied in a cluster configured for GMSA, the following sequence of events take place:
```
Comment (michmike) [inline content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: nit `takes place`

Comment (michmike) [reply content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: oh it is merged. don't worry about it :)

**Pull request merged**